recipes/scikit-data/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "scikit-data" %}
+{% set version = "0.1.0" %}
+{% set sha256 = "bf1a05bb590a202b48bfaf154d875e4e1ab6261ab42e20a2c389b8e1c626c575" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py2k and py<34]
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
```

Comment:
@xmnlab what @synapticarbors is saying here is that, if you don't import the module in `setup.py`, like to get the version, you don't need the run-time dependencies at build-time. This is odd for `pip` users b/c this distinction in the `pip` realm is `install_requires` vs `build_requires`.

I can see in your [setup.py](https://github.com/OpenDataScienceLab/skdata/blob/master/setup.py) that you don't import `scikit-data` itself, so it is save to remove everything but `python` and `setuptools` from the build section.